t/app/controller/reports.t

```diff
@@ -18,6 +18,9 @@ my $body_west_id = $mech->create_body_ok(2504, 'Westminster City Council')->id;
 my $body_fife_id = $mech->create_body_ok(2649, 'Fife Council')->id;
 my $body_slash_id = $mech->create_body_ok(10000, 'Electricity/Gas Council')->id;
 
+my $pothole = $mech->create_contact_ok(body_id => $body_edin_id, category => 'Potholes', email => 'potholes@example.org');
+my $graffiti = $mech->create_contact_ok(body_id => $body_west_id, category => 'Graffiti', email => 'graffiti@example.org');
+my $flytipping = $mech->create_contact_ok(body_id => $body_fife_id, category => 'Flytipping', email => 'flytipping@example.org');
```

Comment:
I'm not sure I understand what adding these lines is doing?

Reply:
Nothing I think, actually, but it seemed odd to be creating reports just underneath that didn't have any matching categories existing for them. I can take them back out.